packages/data-context/src/sources/GitDataSource.ts

```diff
@@ -35,6 +35,9 @@ const GIT_LOG_COMMAND = `git log -1 --pretty="format:%ci %ar %an"`
 const GIT_ROOT_DIR_COMMAND = '--show-toplevel'
 const SIXTY_SECONDS = 60 * 1000
 
+function normalize (file: string) {
+  return file.replace(/\\/g, '/') // normalize \ to /
+}
```

Comment:
i prefer more specific names for utility functions...

```suggestion
function ensurePosixPathSeparators (text: string) {
  return text.replace(/\\/g, '/') // normalize \ to /
}
```

Reply:
Also, since we are on a recent version of Node, you can use `String.prototype.replaceAll`:

```suggestion
function ensurePosixPathSeparators (path: string) {
  return path.replaceAll('\\', '/') // normalize \ to /
}
```